plugins/permission-node/src/integration/createPermissionIntegrationRouter.ts

```diff
@@ -38,8 +38,8 @@ const permissionCriteriaSchema: z.ZodSchema<
   PermissionCriteria<PermissionCondition>
 > = z.lazy(() =>
   z.union([
-    z.object({ anyOf: z.array(permissionCriteriaSchema) }),
-    z.object({ allOf: z.array(permissionCriteriaSchema) }),
+    z.object({ anyOf: z.array(permissionCriteriaSchema).nonempty() }),
```

Comment:
shall we add `strict` as well? Just to avoid passing down things like `{ allOf: [...], anyOf: [...] }`

Reply:
Done in 4448ba4!